pytest_fixtures/rh_cloud.py

```diff
@@ -1,7 +1,25 @@
 import pytest
 from nailgun import entities
 
+from robottelo.config import settings
 from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
+from robottelo.constants import DISTRO_RHEL8
+from robottelo.helpers import add_remote_execution_ssh_key
+from robottelo.rh_cloud_utils import setting_update
+
+
+@pytest.fixture(scope='module')
+def set_rh_cloud_token():
+    """A module-level fixture to set rh cloud token value."""
+    setting_update('rh_cloud_token', str(settings.rh_cloud.token))
+
+
+@pytest.fixture
+def unset_set_cloud_token():
```

Comment:
This is a pretty strange pattern, combined with the module level fixture.  Why not have a single function scoped fixture that sets the value, yields, and then 'unsets' the value. You achieve the same configuration pattern across test cases, without trying to do setup for the next testcase in teardown of the previous.

Reply:
updated the pr.